lerna.json

```diff
@@ -0,0 +1,14 @@
+{
+  "lerna": "0.0.1",
+  "npmClient": "yarn",
+  "useWorkspaces": true,
+  "version": "independent",
+  "command": {
```

Comment:
since yarn does the hard work, publishing becomes the most interesting part of lerna